src/Components/test/testassets/ComponentsApp.Server/Startup.cs

```diff
@@ -17,7 +17,10 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
             services.AddSingleton<CircuitHandler, LoggingCircuitHandler>();
-            services.AddServerSideBlazor();
+            services.AddServerSideBlazor(options =>
+            {
+                options.JSInteropDetailedErrors = true;
```

Comment:
Left over? I don't care if you leave it here tbh as we don't do any exception testing here.

Reply:
It was deliberate - I think this benefits us all.